requirements/constraints.txt

```diff
@@ -24,3 +24,6 @@ diff-cover<4.2.0
 
 # requests has a constraint idna<3 in it, so pinning it unless requests updates the constraint
 idna<3.0
+
+# pyyaml version 6 has a bug where load function throws error so pinning this  
```

Comment:
What was the failure? If the failure was related to `yaml.load` then you can have a look at the PR https://github.com/yaml/pyyaml/pull/561 and the changelog for `pyyaml==6.0` and try to fix the failure first.